frontend/src/metabase/query_builder/components/view/sidebars/QuestionDetailsSidebar.jsx

```diff
@@ -1,43 +1,28 @@
-import React from "react";
-import PropTypes from "prop-types";
-
-import SidebarContent from "metabase/query_builder/components/SidebarContent";
-import QuestionActionButtons from "metabase/questions/components/QuestionActionButtons";
+import React, { useState } from "react";
+import QuestionDetailsSidebarPanel from "metabase/query_builder/components/view/sidebars/QuestionDetailsSidebarPanel";
 import { PLUGIN_MODERATION_COMPONENTS } from "metabase/plugins";
+import { SIDEBAR_VIEWS } from "./constants";
+const { CreateModerationIssuePanel } = PLUGIN_MODERATION_COMPONENTS;
 
-const {
-  active: isPluginActive,
-  ModerationIssueActionMenu,
-} = PLUGIN_MODERATION_COMPONENTS;
-
-function QuestionDetailsSidebar({ question, onOpenModal }) {
-  const canWrite = question && question.canWrite();
+function QuestionSidebarView(props) {
```

Comment:
I think it's handy when the file name matches the main component name inside of it. It makes it easier to find a component and navigate the codebase in general. Do we follow this practice?

Reply:
Oops, ya we should do that. This was a miss on my part when I split `QuestionDetailsSidebar` into two separate components. Fixed!